id,module,name,file,doc,markers,status,message,duration
functional_correctness_test_folder/csn/gpt-5/partial_docstr/test_snippet_290.py::test_case_0,functional_correctness_test_folder.csn.gpt-5.partial_docstr.test_snippet_290,test_case_0,functional_correctness_test_folder/csn/gpt-5/partial_docstr/test_snippet_290.py,,xfail,xfailed,,0.00017420799122191966
functional_correctness_test_folder/csn/gpt-5/partial_docstr/test_snippet_290.py::test_case_1,functional_correctness_test_folder.csn.gpt-5.partial_docstr.test_snippet_290,test_case_1,functional_correctness_test_folder/csn/gpt-5/partial_docstr/test_snippet_290.py,,xfail,xfailed,,8.891697507351637e-05
functional_correctness_test_folder/csn/gpt-5/partial_docstr/test_snippet_290.py::test_case_2,functional_correctness_test_folder.csn.gpt-5.partial_docstr.test_snippet_290,test_case_2,functional_correctness_test_folder/csn/gpt-5/partial_docstr/test_snippet_290.py,,,failed,AttributeError: 'ZMQNameServer' object has no attribute 'running'. Did you mean: '_running'?,8.324999362230301e-05
functional_correctness_test_folder/csn/gpt-5/partial_docstr/test_snippet_290.py::test_case_3,functional_correctness_test_folder.csn.gpt-5.partial_docstr.test_snippet_290,test_case_3,functional_correctness_test_folder/csn/gpt-5/partial_docstr/test_snippet_290.py,,,failed,AttributeError: 'ZMQNameServer' object has no attribute 'running'. Did you mean: '_running'?,8.979201084002852e-05
functional_correctness_test_folder/csn/gpt-5/partial_docstr/test_snippet_290.py::test_case_4,functional_correctness_test_folder.csn.gpt-5.partial_docstr.test_snippet_290,test_case_4,functional_correctness_test_folder/csn/gpt-5/partial_docstr/test_snippet_290.py,,xfail,xfailed,,0.0001050409919116646
